controllers/testdata/deploy/case_2/config.yaml

```diff
@@ -0,0 +1,11 @@
+images:
+  apiServer: quay.io/modh/test-api-server-container:test
+  artifact: quay.io/modh/test-artifact-manager-container:test
+  persistentAgent: quay.io/modh/test-persistenceagent-container:test
+  scheduledWorkflow: quay.io/modh/test-scheduledworkflow-container:test
+  viewerCRD: quay.io/modh/test-viewercontroller-container:test
+  cache: registry.access.redhat.com/ubi8/test-ubi-minimal
+  moveResultsImage: testbusybox
```

Comment:
quay. also, is this an actual thing?

Reply:
this is a a funcitonal test (as denoted by its location), it doesn't matter if this exists, it's not actually used to deploy thing, we're just testing to see if the image config gets repalced